CONTRIBUTING.md

```diff
@@ -222,6 +222,21 @@ Situations when it may be fine to ignore a MiMa issued warning include:
 
 The binary compatibility of the current changes can be checked by running `sbt +mimaReportBinaryIssues`.
 
+## Wire compatibility
+
+Changes to the binary protocol of remoting, cluster and the cluster tools require great care so that it is possible
+to do rolling up and downgrades. 
+
+Often a change to those protocols can require changes in a two version process where 
```

Comment:
```suggestion
Since during a rolling upgrade nodes producing the 'new' format and nodes producing the 'old' format coexist, a change can require a two-phase process: 
```